quranHome/src/main/res/layout/quran_home_view.xml

```diff
@@ -36,30 +22,43 @@
         app:layout_constraintBottom_toBottomOf="@id/tabLayout"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent">
-
-        <FrameLayout
-            android:layout_width="match_parent"
-            android:layout_height="match_parent"
-            android:paddingStart="@dimen/medium_padding"
-            android:paddingEnd="@dimen/medium_padding">
+        app:layout_constraintTop_toTopOf="parent" />
 
-            <TextView
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:layout_gravity="end|center"
-                android:fontFamily="@font/english_regular_light"
-                android:text="@string/search" />
+    <TextView
+        android:id="@+id/searchLabel"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/medium_margin"
+        android:layout_marginEnd="@dimen/medium_margin"
+        android:fontFamily="@font/english_regular_light"
+        android:text="@string/search"
+        app:layout_constraintBottom_toBottomOf="@id/search"
+        app:layout_constraintEnd_toStartOf="@id/search"
+        app:layout_constraintTop_toTopOf="@id/search" />
 
-            <ImageView
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:layout_gravity="start|center"
-                android:src="@drawable/ic_outline_search"
-                tools:ignore="ContentDescription" />
-        </FrameLayout>
+    <ImageView
+        android:id="@+id/search"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/medium_margin"
+        android:layout_marginEnd="@dimen/medium_margin"
+        android:src="@drawable/ic_outline_search"
+        app:layout_constraintBottom_toBottomOf="@id/inputCard"
+        app:layout_constraintEnd_toEndOf="@id/inputCard"
+        app:layout_constraintTop_toTopOf="@id/inputCard"
+        tools:ignore="ContentDescription" />
 
-    </androidx.cardview.widget.CardView>
+    <ImageView
```

Comment:
The hover of Settings icon is not fitted well.

<img src="https://user-images.githubusercontent.com/16861750/94862966-993bff80-0446-11eb-9a41-846f9f5fba31.png" width="30%">
<img src="https://user-images.githubusercontent.com/16861750/94862971-9a6d2c80-0446-11eb-87ff-9a833eb7d017.png" width="30%">

Reply:
i used padding instead of margin because when we click on the edge of this view, search page will open.
do you have any better solution?

Reply:
Good enough, but I was meant the hover, any solution you had, is fine.

Reply:
Fixed 🤟